include/albatross/src/linalg/block_diagonal.hpp

```diff
@@ -157,6 +230,17 @@ inline double BlockDiagonalLDLT::log_determinant() const {
   return output;
 }
 
+inline double BlockDiagonalLDLT::rcond() const {
+  // L1 induced norm is just the maximum absolute column sum.
+  // Therefore the L1 induced norm of a block-diagonal matrix is the
+  // maximum of the L1 induced norms of the individual blocks.
+  double l1_norm = -INFINITY;
```

Comment:
If blocks is empty what should we return? Probably NAN instead of -INFINITY (imagine someone checking `rcond() < 10000`, we'd probably want that to return false?)

Reply:
That's a great point.

Reply:
Added a check for this case.